nixos/modules/services/networking/bee-clef.nix

```diff
@@ -0,0 +1,76 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.bee-clef;
+in {
+  meta = {
+    maintainers = with maintainers; [ attila-lendvai ];
+  };
+
+  ### interface
+
+  options = {
+    services.bee-clef = {
+      enable = mkEnableOption "clef external signer instance for Ethereum Swarm Bee";
+
+      data-dir = mkOption {
+        type = types.nullOr types.str;
+        default = "/var/lib/bee-clef";
+        description = "Data dir for bee-clef.";
+      };
+
+      password-file = mkOption {
+        type = types.nullOr types.str;
+        default = "/var/lib/bee-clef/password";
+        description = "Password file for bee-clef.";
+      };
+    };
+  };
+
+  ### implementation
+
+  config = mkIf cfg.enable {
+    systemd.packages = [ pkgs.bee-clef ];
```

Comment:
Shouldn't it be possible to patch all paths / hard code so this is not necessary?

Reply:
done